<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InjectorWindow</class>
 <widget class="QMainWindow" name="InjectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="InjectorTabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>570</width>
          <height>190</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>5000</width>
          <height>5000</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Software</string>
         </attribute>
         <widget class="QPushButton" name="OpenPushButtonSoftwareTab">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
         <widget class="QListWidget" name="SoftwareListWidgetSoftwareTab">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>561</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="WritePushButtonSoftwareTab">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>10</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Write</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>10</y>
            <width>301</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Address:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="AddressLineEditSoftwareTab">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Signal</string>
         </attribute>
         <widget class="QPushButton" name="SendPushButtonSignalTab">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>10</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>10</y>
            <width>102</width>
            <height>28</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>CPU #:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="CpuSpinBoxSignalTab">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>10</y>
            <width>181</width>
            <height>28</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Segment:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SegmentSpinBoxSignalTab">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>16777215</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="SignalTypeLabelSignalTab">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>41</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Data:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="SignaLineEditSignalTab">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>501</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Memory</string>
         </attribute>
         <widget class="QPushButton" name="ReadPushButtonMemoryTab">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Read</string>
          </property>
         </widget>
         <widget class="QPushButton" name="WritePushButtonMemoryTab">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>40</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Write</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>251</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Address:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="AddressLineEditMemoryTab">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>231</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Data:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DataLineEditMemoryTab">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Status Messages:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="StatusMessagePlainTextEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
